### ISSUE TITLE
Attachment in send mail 

### ISSUE THREAD

**Issue by jolace** [AchrafSoltani/MailgunServiceProvider#2]
Hello, i followed all your instruction and package work great. But i have problem to send attachment in email. I read documentation, took a look in your code...but have a not idea what is problem. 
Can you please check if everything is ok with function, and if function is ok to write small example for send attachments in email?  Tnx

**Comment by achrafsoltani** [comment]
Hello,

Use the same example of sending a message and add an attachment like so 

```
$message = array(
    'from'    => 'Excited User <YOU@YOUR_DOMAIN_NAME>',
    'to'      => 'foo@example.com',
    'cc'      => 'baz@example.com',
    'bcc'     => 'bar@example.com',
    'subject' => 'Hello',
    'text'    => 'Testing some Mailgun awesomeness!'
    'html'    => '<html>HTML version of the body</html>'
), array(
    'attachment' => array('/path/to/file.txt', '/path/to/file.txt')
));

$app['mailgun']->sendMessage($message);
```

**Comment by jolace** [comment]
Yea, i use exactly same example... This is my function :

public function sendCustomMail($app,$parametars)
    {

```
    $message    = array(
        'from'    => 'MYdomain<noreply@mydomain.com>',
        'to'      => 'myemail@domain.com',
        'subject' => 'Subject',
        'html'    => 'Email text'   
    );
    $attachment['attachment'] = array();
    $attachment['attachment'][] = $app['upload_folder'].'avatar.png'; 
    $result = $app['mailgun']->sendMessage($message,$attachment);

    return array('success'=>1);
}
```

At the beginning i thought that maybe file path is problem but when use function is_file() for $app['upload_folder'].'avatar.png'  return true so file path is ok.

I am not sure what is problem.

**Comment by jolace** [comment]
In your file Client.php  on line 37 you call 

public function sendMessage($postData, $postFiles = array())
{
     return parent::sendMessage($this->workingDomain, $postData, **$postFiles = array()** );
}

There you overwrite attachment with empty array so that was reason. You will need to change it to 

public function sendMessage($postData, $postFiles = array())
{
     return parent::sendMessage($this->workingDomain, $postData, **$postFiles** );
}

Thanks

**Comment by achrafsoltani** [comment]
fixed in v1.0.5, please run composer update
